students/tammyd/lesson02/series.py

```diff
@@ -31,6 +33,8 @@ def lucas(n):
 #I am having a hard time understanding this part. I will ask during class. 
 
 """This function called sum_series has one required parameter and two optional parameters. The required parameter determines which element in the series to print. The two optional parameters have default values of 0 and 1 and determine the first two values for the series to be produced."""
+"""
+
 def sum_series(required, *args, **kwargs):
 
     if x == 0 and y == 1:
```

Comment:
ahh -- this is backwards from what we want -- you want to write a generic one, where you specify x and y, and then you can write a specific one, where x and y are fixed.

See the Solutions.